cdc/cdc/processor/manager.go

```diff
@@ -155,6 +155,14 @@ func (m *Manager) sendCommand(tp commandTp, payload interface{}) chan struct{} {
 	return cmd.done
 }
 
+// Close closes all processors.
+// Note: This method must not be called with `Tick`. Please be careful.
+func (m *Manager) Close() {
```

Comment:
Suggest to name as `SyncClose`, and reuse in handling of `commandTpClose` command.